### PR TITLE
Drop selenium_sessions table

### PR DESCRIPTION
**Pull request by albertchae**
There are only 3 entries in prod from 2021, so it seems safe to wipe that data

### CODE REVIEW

**Comment by albertchae** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3695** <a href="https://app.graphite.dev/github/pr/hackclub/bank/3695" target="_blank">Graphite</a> 
    * **PR #3741** <a href="https://app.graphite.dev/github/pr/hackclub/bank/3741" target="_blank">Graphite</a>  👈
      * **PR #3742** <a href="https://app.graphite.dev/github/pr/hackclub/bank/3742" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/hackclub/bank/3741?utm_source=stack-comment).

File: db/schema.rb [hunk 1]
```diff
@@ -1269,19 +1269,12 @@
     t.text "url_hash"
     t.text "last4_ciphertext"
     t.datetime "canceled_at"
-    t.boolean "migrated_from_legacy_stripe_account"
+    t.boolean "migrated_from_legacy_stripe_account", default: false
```
Comment (maxwofford) [inline db/schema.rb]: Did you mean to add this?

**Review by maxwofford — APPROVED** [review]

File: db/schema.rb (hunk 1 above)
Comment (albertchae) [reply db/schema.rb]: oops, no i didn't. Removed

Comment (albertchae) [reply db/schema.rb]: This comment made me look into this a bit, and I think the code comments are out of the date with the db schema https://github.com/hackclub/bank/pull/3772

**Pull request merged**